metrics.go

```diff
@@ -68,27 +68,27 @@ type LevelMetrics struct {
 	FillFactor float64
 	// The level's compensated fill factor. See candidateLevelInfo.
 	CompensatedFillFactor float64
-	// The number of incoming bytes from other levels read during
+	// The number of incoming bytes from other levels' sstables read during
 	// compactions. This excludes bytes moved and bytes ingested. For L0 this is
 	// the bytes written to the WAL.
-	BytesIn uint64
-	// The number of bytes ingested. The sibling metric for tables is
+	TableBytesIn uint64
+	// The number of sstable bytes ingested. The sibling metric for tables is
 	// TablesIngested.
-	BytesIngested uint64
-	// The number of bytes moved into the level by a "move" compaction. The
-	// sibling metric for tables is TablesMoved.
-	BytesMoved uint64
+	TableBytesIngested uint64
+	// The number of sstable bytes moved into the level by a "move" compaction.
+	// The sibling metric for tables is TablesMoved.
+	TableBytesMoved uint64
 	// The number of bytes read for compactions at the level. This includes bytes
 	// read from other levels (BytesIn), as well as bytes read for the level.
-	BytesRead uint64
-	// The number of bytes written during compactions. The sibling
-	// metric for tables is TablesCompacted. This metric may be summed
-	// with BytesFlushed to compute the total bytes written for the level.
-	BytesCompacted uint64
-	// The number of bytes written during flushes. The sibling
-	// metrics for tables is TablesFlushed. This metric is always
-	// zero for all levels other than L0.
-	BytesFlushed uint64
+	TableBytesRead uint64
+	// The number of bytes written to sstables during compactions. The sibling
+	// metric for tables is TablesCompacted. This metric may be summed with
+	// BytesFlushed to compute the total bytes written for the level.
+	TableBytesCompacted uint64
+	// The number of bytes written to sstables during flushes. The sibling
+	// metrics for tables is TablesFlushed. This metric is always zero for all
```

Comment:
Was this meant to say `BlobBytesCompacted` instead of `BytesFlushed`? We are missing a `BlobBytesCompacted` metric